docs/navigation.yml

```diff
@@ -1,13 +1,14 @@
 arrange:
-    - index.md
-    - installation
-    - introduction
-    - model
-    - cookbook
-    - functional
-    - frontend
-    - administration
-    - frontend-api
-    - extensibility
-    - automated-testing
-    - contributing
+  - index.md
+  - installation
+  - introduction
+  - model
+  - cookbook
```

Comment:
please use 4 spaces indent in all yml files

Reply:
Not possible with usage of Prettier. We need to figure out how to combine IDE setup which you use on BE with our setup on FE side.

Reply:
Should be resolved now